mappings/net/minecraft/recipe/IRecipeFinderPopulator.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS auf net/minecraft/recipe/IRecipeFinderPopulator
+	METHOD a populateRecipeFinder (Lard;)V
```

Comment:
RecipeFinderPopulator?

... Populator? RecipeFinder? Can we find better names for these?

Reply:
I dunno. You can use it to find out whether or not a single recipe is craftable with a given collection of items, but only after you've populated it with those items, either by directly adding items to it or by passing it to an object that presents itself as a finder-populator. Also considered RecipeMatcher and matcher-populator, but I put that aside because individual recipes already match themselves against a CraftingInventory (or furnace-type block entity, etc).

Reply:
RecipeInputProvider, now.